modules/menu/dynamic_menu.c

```diff
@@ -12,76 +12,49 @@
 static int set_audio_bitrate(struct re_printf *pf, void *arg)
 {
 	struct cmd_arg *carg = arg;
-	struct ua *ua = carg->data ? carg->data : menu_uacur();
-	struct call *call;
+	struct call *call = menu_getcall(pf);
```

Comment:
we should keep this logic:

1. if carg->data is set, use ua from there
2. if carg->data is NULL, lookup ua from current call

Reply:
Oh, I see. Did'nt saw how cmd_process_long() is used in mqtt. In ctrl_tcp and ctrl_dbus we should do this in the same way. 

But here is a problem. The parameter is a void*. And you allow all commands. Suppose one writes a command "do_foo" and wants to get a foo* in the data parameter and maybe he writes also an ui module which correctly passes a foo* in the data parameter. Then a user may also call do_foo by means of mqtt and specify the accountaor. Then ua* is cast to foo*.

Reply:
For now I'll keep the logic.